manifests/fastd.pp

```diff
@@ -8,6 +8,7 @@
                      , $fastd_port
 
                      , $fastd_peers_git
+                     , $fastd_verify = ''
```

Comment:
Because this could be empty, contain 'true' or a path to a script that verifies the connection attempt